test/scenarios/ynEIGEN/ynLSDeWithdrawals.t.sol

```diff
@@ -388,13 +317,20 @@ contract ynLSDeWithdrawalsTest is ynLSDeScenarioBaseTest {
         uint256 _ynEigenWOETHBalanceBefore = yneigen.assets(chainAddresses.lsd.WOETH_ADDRESS);
         uint256 _ynEigenSFRXETHBalanceBefore = yneigen.assets(chainAddresses.lsd.SFRXETH_ADDRESS);
 
-        vm.prank(actors.ops.WITHDRAWAL_MANAGER);
-        eigenStrategyManager.processPrincipalWithdrawals(_actions);
+        {
+            uint256 _redemptionVaultWSTETHBalanceBefore = redemptionAssetsVault.balances(chainAddresses.lsd.WSTETH_ADDRESS);
```

Comment:
the main thing about these tests is that they can't assume previous state being 0

so it needs to use deltas before and after